src/app/dashboard/dashboard.component.ts

```diff
@@ -19,12 +19,18 @@ export class DashboardComponent implements OnInit{
   ]
 
 
-  // ngOnInit method
+  /**
+   * ngOnInit method
+   * @returns void
+   *  */ 
   ngOnInit(): void{
     this.totalBalanceCalculate();
   }
 
-  // Calculate total balance, owe amount and owed amount
+  /**
+   * Calculate total balance, owe amount and owed amount
+   * @returns void
```

Comment:
No need to add returns if it is void. its okay in this PR. from next PR you can take this consideration.

Reply:
ok di